src/State/ObjectMapper/ObjectMapper.php

```diff
@@ -17,28 +17,44 @@
 use Symfony\Component\ObjectMapper\ObjectMapperAwareInterface;
 use Symfony\Component\ObjectMapper\ObjectMapperInterface;
 
-final class ObjectMapper implements ObjectMapperInterface
-{
-    public function __construct(private ObjectMapperInterface $decorated)
+if (class_exists(ObjectMapperAwareInterface::class)) {
```

Comment:
I think the `if` should be the other way around. The way it is right now you're including `ObjectMapperAwareInterface` if it *doesn't* exist and not if it does, which obviously doesn't make sense.

Reply:
Indeed! @soyuka fix PR incoming in https://github.com/api-platform/core/pull/7539